### PR TITLE
Musica estilo mariachi

### PR DESCRIPTION
**Pull request by knaguilar**

### CODE REVIEW

**Comment by knaguilar** [comment]
Please review this, @casey-collab

File: espanol/musica/grupo_mariachi.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Y fue entonces cuando apareció un grupo de Mariachi, listo para sorprender a la familia con un poco de música.
```
Comment (casey-collab) [inline espanol/musica/grupo_mariachi.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by knaguilar** [comment]
Please review this, @casey-collab I have made the necessary changes

**Comment by casey-collab** [comment]
Looks great! :smiley_cat:

**Pull request merged**